examples/rubber-ball.html

```diff
@@ -27,22 +27,20 @@
             autostart: true
           }).appendTo(document.body);
 
-          Two.Resoultion = 32;
-
           var delta = new Two.Vector();
           var mouse = new Two.Vector();
           var drag = 0.33;
           var radius = 50;
 
-          var shadow = two.makeCircle(two.width / 2, two.height / 2, radius);
+          var shadow = two.makeCircle(two.width / 2, two.height / 2, radius, 32);
```

Comment:
Same as with morph.html, circle resolution is 4 unless otherwise specified as is now done here.